src/components/ads/AdClickThroughButton.ts

```diff
@@ -27,16 +27,16 @@ export class AdClickThroughButton extends StateReceiverMixin(LinkButton, ['playe
     constructor() {
         super({ template: template() });
 
-        this.disabled = true;
-        this.style.display = 'none';
-
         this._upgradeProperty('clickThrough');
         this._upgradeProperty('player');
     }
 
     override connectedCallback(): void {
         super.connectedCallback();
         this._updateFromPlayer();
+
+        this.disabled = true;
+        this.style.display = 'none';
```

Comment:
If the `clickThrough` property was already set before, we shouldn't reset these properties:
```suggestion
        if (this.clickThrough == null) {
            this.disabled = true;
            this.style.display = 'none';
        }
```